packages/react-components/src/components/MicrophoneButton.tsx

```diff
@@ -240,6 +262,24 @@ export const MicrophoneButton = (props: MicrophoneButtonProps): JSX.Element => {
             iconName: props.checked ? 'SplitButtonPrimaryActionMicUnmuted' : 'SplitButtonPrimaryActionMicMuted',
             styles: { root: { lineHeight: 0 } }
           }
+        },
+        /* @conditional-compile-remove(DNS) */
+        {
+          key: 'microphoneDNSToggle',
+          onRender: () => {
+            return (
+              <Toggle
```

Comment:
Have we tested here what the narrator says? we might want to re-use that label to make accessible. More of a general note: I think we might want to transition from using explicit aria labels and have them be the label of the control and look into using `aria-description` for more detailed information about a control. 

This to say that we should have the aria label match the visual label to short circuit these bugs

Reply:
Done, added the announcer strings and switch on toggle change